script/glam/test/test_glean_firefox_desktop_glam_nightly.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# generate sql for checking into the repository and for testing the workflow
+
+set -e
+
+project=${PROJECT:-glam-fenix-dev}
+skip_generate=${SKIP_GENERATE:-false}
+skip_daily=${SKIP_DAILY:-false}
+generate_only=${GENERATE_ONLY:-false}
+# NOTE: there are three app_ids that we must look at for historical context. For
+# the purpose of this script, it is sufficient to look only at what is currently
+# "firefox desktop". We must have at least one table scalar/histogram tables for
+# each of the referenced tables in the view. We'll keep all pings for
+# firefox_desktop, and only the metrics ping for the others.
+app_ids=(
+    "firefox_desktop"
+)
+logical_app_id="firefox_desktop_glam_nightly"
```

Comment:
I think this file is identical to the fenix one except for these two lines above right? Can we use the same test file for both and just have an if statement here depending and set the values of `app_ids` and `local_app_id` base on an additional input param that specifies desktop or fenix?